<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PMT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabs">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <attribute name="title">
         <string>Projects</string>
        </attribute>
        <widget class="QPushButton" name="CPbutton">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>10</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Create Project</string>
         </property>
        </widget>
        <widget class="QComboBox" name="SDcomboBox">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>50</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>theOtherOnesGames</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>theOtherOnesFilms</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>55</y>
           <width>191</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Select your department:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Project Name:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="DPbutton">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>460</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Delete Project</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="PNlineEdit">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>10</y>
           <width>251</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QTreeView" name="DirTreeView">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>591</width>
           <height>351</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Assets</string>
        </attribute>
        <widget class="QComboBox" name="SPcomboBox">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>60</y>
           <width>271</width>
           <height>31</height>
          </rect>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Hi</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>371</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Create Asset Type (Select all that apply):</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Select Project:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="DAbutton">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>449</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Delete Asset</string>
         </property>
        </widget>
        <widget class="QPushButton" name="OAbutton">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>449</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Open Asset</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Name Asset:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="CAbutton">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>130</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Create Asset(s)</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="MeshCheckBox">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>100</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Mesh</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="RigCheckBox">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>100</y>
           <width>57</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Rig</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="AnimCheckBox">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>100</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Animation</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="NAlineEdit">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>130</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QTreeView" name="assetTreeView">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>180</y>
           <width>591</width>
           <height>261</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Select Deparment:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="SDAcomboBox">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>20</y>
           <width>241</width>
           <height>31</height>
          </rect>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Hi</string>
         </property>
         <item>
          <property name="text">
           <string>theOtherOnesGames</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>theOtherOnesFilms</string>
          </property>
         </item>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
